orion/pkg/domains/location.go

```diff
@@ -10,16 +10,17 @@ import (
 var TABLE_LOCATIONS = "locations"
 
 type Location struct {
-	Id        uint         `json:"id"`
-	CreatedAt time.Time    `json:"-" db:"created_at"`
-	UpdatedAt time.Time    `json:"-" db:"updated_at"`
-	DeletedAt sql.NullTime `json:"-" db:"deleted_at"`
-	LocId     string       `json:"locId" db:"loc_id"`
-	Street    string       `json:"street"`
-	City      string       `json:"city"`
-	State     string       `json:"state"`
-	Zipcode   string       `json:"zipcode"`
-	Room      NullString   `json:"room"`
+	Id          uint
```

Comment:
keep the json stuff from before. we want every field to have a `json:"..."` tag.